components/datePicker/ru_th.ts

```diff
@@ -33,4 +33,4 @@ const thai_th: Locale = {
   ],
```

Comment:
The short names of months are inconsistent. Some of them have a dot at the end, some do not. The locale objects stored in the `react-date-object` package, which is the basis for this solution, do not have dots at the short month names.
Short names of the days of the week should also not have dots at the end.

Reply:
done